.travis.yml

```diff
@@ -1,4 +1,10 @@
 services:
   - docker
+language: python
+python:
+  - "3.7"
+cache: pip
 script:
   - ./test.sh
+  - pip install -r requirements.txt
```

Comment:
Travis does this by default: https://docs.travis-ci.com/user/languages/python/#what-this-guide-covers
```suggestion
```